addons/drag-drop/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Drag and drop files",
-  "description": "Lets you drag images and sounds from your file explorer into the sprites pane or costume/sound list. You can also drag text files into lists or \"ask and wait\" prompts.",
+  "name": "File drag and drop",
```

Comment:
Should be kept imo. "drag and drop" is still a verb.

Reply:
In `Drag and drop files`, "drag and drop" is used as a verb so I agree with this change (and disagree with apple502j)